wordpress-mobile/WordPress-Android
# Jetpack App (Basics): Prologue screen

SubTask: #14302

PS: Don't be alarmed with the PR size, its because of `bg_stars_background_large.xml ` ( 2,436 lines)

This PR updates the prologue screen for the Jetpack app. 

![device-2021-05-04-135123](https://user-images.githubusercontent.com/1405144/116977564-34989a00-ace0-11eb-96ba-6b4f78bd9a17.png)

To test:

Jetpack app
- Install and launch Jetpack app. 
- Notice that the prologue screen for the Jetpack app is displayed.
(Check for both light and dark mode.)

WordPress app 
- Install WordPress app. 
- Notice that the prologue screen for WordPress app is displayed.

Merge Instructions

1. Wait for design review
2. Remove Not Ready for Merge label
3. Merge the PR


## Regression Notes
1. Potential unintended areas of impact
WordPress app

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Tested prologue screen for WordPress app manually.

3. What automated tests I added (or what prevented me from doing so)
Added tests for prologue screen for Jetpack app.

PR submission checklist:
